src/utils/i18n/translations/en-US.json

```diff
@@ -13,6 +13,12 @@
   "pagination.limit.suffix": "records per page",
   "password.hiddenPassword": "Hide Password",
   "password.viewPassword": "View Password",
+  "password.strength.label": "Password strength: ",
+  "password.strength.level.very-weak": "Very Weak",
+  "password.strength.level.weak": "Weak",
+  "password.strength.level.fair": "Fair",
+  "password.strength.level.strong": "Strong",
+  "password.strength.level.very-strong": "Very Strong",
   "password.generateNew": "Generate new password",
   "password.poor": "Poor!",
```

Comment:
please remove the old password strength translations

Reply:
done